Formula/tensorflow_c.rb

```diff
@@ -0,0 +1,42 @@
+class TensorflowC < Formula
+  desc "C interface for Google's OS library for Machine Intelligence"
+  homepage "https://www.tensorflow.org/"
+  url "https://github.com/tensorflow/tensorflow", :using => :git,
+    :tag => "v1.0.0", :revision => "07bb8ea2379bd459832b23951fb20ec47f3fdbd4"
+
+  depends_on "bazel" => :build
+  depends_on "pkg-config" => :run
+
+  def install
+    system 'echo "\n\n\n\n\n\n\n\n\n" | ./configure'
```

Comment:
This one is very sad. Tensorflow `./configure` is not an autotools like thing. It will expect you to hit return 9 times to accept to default to 9 questions. There is not provision for non interactive run.

Alternatively, I could do a `yes ''| ./configure` but `yes` would fail systematically with a borken pipe when configure ends and it would get tricky to detect a ./configure fail.

If you have a better idea, I'm all hears.